src/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Wbpack Exercise</title>
+</head>
+<body>
+    <div id="todoContainer">
+        <div class="headerContainer">
+            <h1 class="header">Today's To Do</h1>
+            <div class="btnContainer">
+                <button class="refreshBtn">&#8634</button>
+            </div>
+        </div>
+
+        <div class="addList">
+            <input type="text" placeholder="Add to your list..." required>
+            <button type="submit" class="addListBtn">&#10550</button>
+        </div>
+
+        <!-- <div class="checkContainer">
+            <div class="inputContainer">
+                <input id="checkBox" type="checkbox"  required>
+                <p id="label">Cook afternoon food</p>
+            </div>
+            <div class="btnContainer">
+                <button class="btnDelete">&#65049</button>    
+            </div>
+        </div> -->
+
+        <div id="todoList">
+
+        </div>
+        <div class="clearListBtnContainer">
+            <button class="clearListBtn" type="button">Clear all completed</button>
+        </div>
+    </div>
+    <script src="index.js"></script>
```

Comment:
- [ ] [OPTIONAL] you don't have to import  the index.js file in the index.html,  webpack will bundle our JavaScript and CSS files and will add them automatically since we specified the entry point